internal/cli/atlas/alerts/settings/list.go

```diff
@@ -50,9 +50,11 @@ var settingsListTemplate = `ID	TYPE	ENABLED{{range .Results}}
 
 func (opts *ListOpts) Run() error {
 	params := &admin.ListAlertConfigurationsApiParams{
-		GroupId:      opts.ConfigProjectID(),
-		ItemsPerPage: &opts.ItemsPerPage,
-		PageNum:      &opts.PageNum,
+		GroupId: opts.ConfigProjectID(),
+		PageNum: &opts.PageNum,
+	}
+	if opts.ItemsPerPage > 0 {
+		params.ItemsPerPage = &opts.ItemsPerPage
```

Comment:
Before the change:
`GET /api/atlas/v2/groups/641deaf951ed5c679239b1a1/alertConfigs?includeCount=true&itemsPerPage=0&pageNum=0 HTTP/1.1`

After the change:
`GET /api/atlas/v2/groups/641deaf951ed5c679239b1a1/alertConfigs?includeCount=true&itemsPerPage=100&pageNum=0 HTTP/1.1`

Reply:
@wtrocki Can we handle this in SDK to send default values instead of 0?

Reply:
Before the change: should be probably After the change:? 

See: https://github.com/mongodb/mongodb-atlas-cli/pull/2032/files#diff-191504041900bd242a9d7945d7e9ee81863889f0a8bba32a0e1cfc34879d939fR96

It clearly sets 0. If you wish different value we should change that